src/ProjectNeon/Assets/Features/Battle/Cards/CardResolutionZone.cs

```diff
@@ -9,7 +9,8 @@ public class CardResolutionZone : ScriptableObject
     [SerializeField] private CardPlayZone physicalZone;
     [SerializeField] private CardPlayZone playedDiscardZone;
     [SerializeField] private GameEvent onFinished;
-    
+    public int SpellHitsOnLastCard { get; set; }
```

Comment:
This does not belong here. 

Realistically, we shouldn't be changing this class, basically ever.

Reply:
@SilasReinagel Okay. How about putting it onto `BattleState` then?